### PR TITLE
FIX: Narrowed down

### PR DESCRIPTION
**Pull request by larsoner**

### CODE REVIEW

**Comment by larsoner** [comment]
Not 100% done yet, still need to fix a handful of subjects. I also haven't looked at any of the reports to see if they make sense.

But @pettetmw you should be able to get through the `fetch_raw=True` step at this point, which takes some time. Feel free to run it locally to see if that works at least!

**Comment by larsoner** [comment]
Note that this requires some bug fixes in https://github.com/LABSN/mnefun/pull/336

File: badbaby/processing/badbaby.yml [hunk 1]
```diff
@@ -233,8 +237,10 @@ preprocessing:
     proj_meg: combined
 
 epoching:
-  tmin: 0
-  tmax: 1
+  # These limits should work for AM and ASSR, IDs will need a separate
+  # epoching step anyway
+  tmin: -0.2
+  tmax: 1.3
```
Comment (ktavabi) [inline badbaby/processing/badbaby.yml:243]: this window will not work for oddball sim

File: badbaby/processing/run_mnefun.py [hunk 2]
```diff
@@ -19,9 +19,11 @@
 - 223a: Preproc (Only 13/15 good ECG epochs found)
 
 About half the time their HPI was no good, so throw them out.
-"""
+"""  # noqa: E501
 
-# TODO: @larsoner sort out combined channel SSP & determine whether `cont_as_esss` is beneficial @larsoner
+# TODO:
+# 1. Deal with failures
+# 2. Rerun all epochs and covariance with updated tmin/tmax
```
Comment (ktavabi) [inline badbaby/processing/run_mnefun.py:26]: the global epoching window will not work for oddball

File: badbaby/processing/badbaby.yml (hunk 1 above)
Comment (larsoner) [reply badbaby/processing/badbaby.yml:243]: We can always `.crop` afterward (and even re-baseline), so I think it might be okay (?)

Comment (ktavabi) [reply badbaby/processing/badbaby.yml:243]: Right on! let's do that.

**Pull request merged**